unyt/tests/test_unyt_array.py

```diff
@@ -1544,15 +1553,13 @@ def test_modified_unit_division():
     reg1 = UnitRegistry()
     reg2 = UnitRegistry()
 
-    # this mocks comoving coordinates without going through the trouble
-    # of setting up a fake cosmological dataset
-    reg1.modify('m', 50)
+    reg1.modify('g', 50)
 
-    a = unyt_quantity(3, 'm', registry=reg1)
-    b = unyt_quantity(3, 'm', registry=reg2)
+    a = unyt_quantity(3, 'g', registry=reg1)
+    b = unyt_quantity(3, 'g', registry=reg2)
 
     ret = a/b
-    assert ret == 0.5
+    assert ret == 50000.
```

Comment:
I think I'm being dumb, but is this correct?

Reply:
Ah, it's because now the `base_value` of gram is .001, as it should be. Apologies for the dumbness.

I think this is ready to merge? Any objections?

Reply:
I don't have any, but I wrote the PR, so...

We should maybe ping @MatthewTurk @chummels @brittonsmith @Xarthisius

Reply:
@MatthewTurk @chummels @brittonsmith @Xarthisius sorry to ping you again, but I just updated the title and description of the PR, so if you started looking it in the past few minutes, you should read the title and description again--the changes are pretty important.

Reply:
Asking a bunch of astronomers to abandon cgs?! That's bold! :D 
Sounds good to me!

Reply:
When we decide to make `yt` depend on `unyt` in yt 4.0, we should be able to maintain cgs as the default in `yt` with only a little bit of trouble.

Reply:
Yup, another thing we might eventually do is make the default unit system used by e.g. `in_base` and friends is configurable.

Reply:
> yt will potentially use unyt as a dependency and it will effectively make yt's units to default to MKS). Is this right?

I'd replace potentially with almost definitely :)

> since most of the user-base are astronomers who will want to continue having cgs as their default unit system.

Yup. I'm certain the astrophysics simulation domain context will default to CGS units.